algolia/search/client_interface.go

```diff
@@ -9,7 +9,9 @@ import (
 type ClientInterface interface {
 	// Misc
 	WaitTask(taskID int64, opts ...interface{}) error
+	WaitRecommendTask(taskID int64, opts ...interface{}) error
```

Comment:
This is a breaking change. It should have been done in a major. Client implementing the interface will now receive a compilation error because their implementation is lacking `WaitRecommendTask` & `GetRecommendStatus`.

Reply:
Worth asking the DX team imo. Not sure who is responsible for this package anymore.

Reply:
We don't really know either. I'll ask in #help-apiclients.